pkg/volume/csi/nodeinfomanager/nodeinfomanager.go

```diff
@@ -29,7 +29,7 @@ import (
 	"time"
 
 	"k8s.io/api/core/v1"
-	storagev1beta1 "k8s.io/api/storage/v1beta1"
+	storagev1 "k8s.io/api/storage/v1"
```

Comment:
Many code changes that replacing StorageV1Beta1 with StoreageV1 was missing. Need to check whether it is ok.

Reply:
Does not affect density test. File an issue

Reply:
https://github.com/CentaurusInfra/arktos/issues/1099